assets/dist/mix-manifest.json

```diff
@@ -45,7 +45,7 @@
     "/styles/login.css": "/styles/login.css?id=176a7ef67256eaaac7dc",
     "/styles/glossary-tooltip.css": "/styles/glossary-tooltip.css?id=d56ddc5bce205919dab7",
     "/scripts/eventsource.polyfill.js": "/scripts/eventsource.polyfill.js?id=7d3bb72e2588790aa9f2",
-    "/scripts/paged.polyfill.js": "/scripts/paged.polyfill.js?id=2ea9287e7b5ed3e6bf8a",
+    "/scripts/paged.polyfill.js": "/scripts/paged.polyfill.js?id=a908de1deb6bcd8e0576",
```

Comment:
We never ran 'npm run build' after merging: https://github.com/pressbooks/pressbooks/pull/2320